pcdsdevices/fms.py

```diff
@@ -27,171 +28,281 @@ class Setra5000(Device):
 
 
 class PDU_Temp4(Device):
-
     """ For racks with 2 PDUs, with total of 4 sensors for monitoring purposes """
 
-    Sensor1 = FCpt(EpicsSignalRO, '{self.prefix}:{self.elevation[0]}:Sensor:1:GetTempValue', kind='hinted')
-    Sensor2 = FCpt(EpicsSignalRO, '{self.prefix}:{self.elevation[1]}:Sensor:2:GetTempValue', kind='hinted')
-    Sensor3 = FCpt(EpicsSignalRO, '{self.prefix}:{self.elevation[2]}:Sensor:1:GetTempValue', kind='hinted')
-    Sensor4 = FCpt(EpicsSignalRO, '{self.prefix}:{self.elevation[3]}:Sensor:2:GetTempValue', kind='hinted')
+    sensor1 = FCpt(EpicsSignalRO,
+                   '{self.prefix}:{self.elevation[0]}:Sensor:1:GetTempValue',
```

Comment:
Could significantly shorten these if you want:
```suggestion
                   '{prefix}:{elevation[0]}:Sensor:1:GetTempValue',
```